src/main/kotlin/uk/gov/dluhc/printapi/mapper/PrintDetailsToPrintRequestMapper.kt

```diff
@@ -0,0 +1,53 @@
+package uk.gov.dluhc.printapi.mapper
+
+import org.mapstruct.Mapper
+import org.mapstruct.Mapping
+import uk.gov.dluhc.printapi.database.entity.PrintDetails
+import uk.gov.dluhc.printapi.printprovider.models.PrintRequest
+import java.time.OffsetDateTime
+import java.util.Date
+
+@Mapper
+abstract class PrintDetailsToPrintRequestMapper {
+
+    @Mapping(source = "details.vacNumber", target = "cardNumber")
+    @Mapping(source = "details.vacVersion", target = "cardVersion")
+    @Mapping(source = "details.firstName", target = "cardFirstname")
+    @Mapping(source = "details.middleNames", target = "cardMiddlenames")
+    @Mapping(source = "details.surname", target = "cardSurname")
+    @Mapping(source = "details.delivery.addressee", target = "deliveryName")
+    @Mapping(source = "details.delivery.address.property", target = "deliverypProperty")
```

Comment:
Seems like there is a typo with the `deliveryProperty`: `deliverypProperty`.

Reply:
Yes, there are several in the YAML spec that the target is generated from - agreed with @NathanRussellValtech that these typos would not be fixed as part of this PR.

deliverypProperty, eroDeliverypPropertyCy, erodDeliveryAreaCy  & cardMiddlenames-